app/views/recommended_links/_update_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for(recommended_link) do |f| %>
```

Comment:
@Davidslv I thought about doing this but the edit form omits a couple of fields and I thought it would be clearer to separate out the partial. What do you think?